generative_ai/rag.py

```diff
@@ -342,20 +354,29 @@ def quickstart(
     # TODO(developer): Update and un-comment below lines
     # project_id = "PROJECT_ID"
     # display_name = "test_corpus"
-    # paths = ["https://drive.google.com/file/123", "gs://my_bucket/my_files_dir"]  # Supports Google Cloud Storage and Google Drive Links
+    # paths = ["https://drive.google.com/file/d/123", "gs://my_bucket/my_files_dir"]  # Supports Google Cloud Storage and Google Drive Links
 
     # Initialize Vertex AI API once per session
     vertexai.init(project=project_id, location="us-central1")
 
     # Create RagCorpus
-    rag_corpus = rag.create_corpus(display_name=display_name)
+    # Configure embedding model, for example "textembedding-gecko@003".
+    embedding_model_config = rag.EmbeddingModelConfig(
+        publisher_model="publishers/google/models/textembedding-gecko@003"
```

Comment:
Can we update this to textembedding-004?

Reply:
done